packages/formation-react/src/components/PromoBanner/PromoBanner.stories.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+import PromoBanner, { PROMO_BANNER_TYPES } from './PromoBanner';
+
+export default {
+  title: 'Library/PromoBanner',
+  component: PromoBanner,
+};
+
+const Template = args => {
+  const [showBanner, setShowBanner] = useState(true);
+  const onClose = () => setShowBanner(false);
+  return showBanner ? (
+    <PromoBanner {...args} onClose={onClose} />
+  ) : (
+    <button onClick={() => setShowBanner(true)}>Show Banner</button>
+  );
```

Comment:
How do you feel about adding a little something to make it clearer _which_ banner is shown on the docs page?
```suggestion
  return showBanner ? (
    <>
      <PromoBanner {...args} onClose={onClose} />
      <p>See the open banner on the bottom of the page.</p>
    </>  
  ) : (
    <button onClick={() => setShowBanner(true)}>Show Banner</button>
  );
```

Reply:
Nice touch!